imports/plugins/included/payments-braintree/lib/collections/schemas/braintree.js

```diff
@@ -31,6 +31,14 @@ export const BraintreePackageConfig = new SimpleSchema([
       type: String,
       label: "Private Key",
       optional: false
+    },
+    "settings.reaction-braintree.support": {
+      type: Array,
+      label: "Payment provider supported methods"
+    },
+    "settings.reaction-braintree.support.$": {
+      type: String,
+      allowedValues: ["authorize", "de-authorize", "capture", "refund"]
```

Comment:
This/these need to be standardized either Capitalized or not capitalized. And when displayed to the customer they should be capitalized.